com.adsbynimbus.nimbus/Runtime/Scripts/Nimbus.Internal/Interceptor/SkAdNetworkIOS.cs

```diff
@@ -34,22 +36,32 @@ public string GetProviderRtbDataFromNativeSDK(AdUnitType type, bool isFullScreen
 			return "";
 		}
 
-		public BidRequest ModifyRequest(BidRequest bidRequest, string data) {
+		public BidRequestDelta ModifyRequest(BidRequest bidRequest, string data) {
```

Comment:
ModifyRequest has public, private, and internal visibility depending on the file; it would be good to keep these consistent.